cnf-certification-test/platform/suite.go

```diff
@@ -82,3 +92,153 @@ func testContainersFsDiff(env *provider.TestEnvironment) {
 	gomega.Expect(badContainers).To(gomega.BeNil())
 	gomega.Expect(errContainers).To(gomega.BeNil())
 }
+
+//nolint:funlen
+func testTainted(env *provider.TestEnvironment) {
+	var taintedNodes []string
+	var errNodes []string
+
+	// Loop through the debug pods that are tied to each node.
+	for _, dp := range env.DebugPods {
+		// Build a NodeTainted tester object.
+		tester := nodetainted.NewNodeTaintedTester(common.DefaultTimeout, ocpclient.NewOcpClient(), ocpclient.Context{
+			Namespace:     dp.Namespace,
+			Podname:       dp.Name,
+			Containername: dp.Spec.Containers[0].Name,
+		})
+		taintInfo, err := tester.GetKernelTaintInfo()
+		if err != nil {
+			logrus.Error("failed to retrieve kernel taint information from debug pod/host")
+			tnf.ClaimFilePrintf("failed to retrieve kernel taint information from debug pod/host")
+			return
+		}
+		tnf.ClaimFilePrintf(fmt.Sprintf("Pod: %s taintInfo retrieved: %s", dp.Name, taintInfo))
```

Comment:
better to log namespace as well so that Pod is unique.

Reply:
Fixed.